envi/archs/i386/emu.py

```diff
@@ -1480,6 +1497,9 @@ def i_rcr(self, op):
         src = self.getOperValue(op, 1)
 
         src = src & 0x1f
+        if dsize < 3:
+            src %= (0, 9, 17)[dsize]
```

Comment:
Same here, maybe just use shiftmask?

Reply:
it's slower, that's why i hesitate.  
i'm tempted to rip out shiftMask/RC altogether and just inline them, since they're only used a couple places, and while emulating, the extra call costs.  then again, probably not enough for an analysis platform?